fast64_internal/sm64/sm64_objects.py

```diff
@@ -1936,6 +1995,9 @@ def sm64_obj_register():
     bpy.types.Object.sm64_obj_use_act5 = bpy.props.BoolProperty(name="Act 5", default=True)
     bpy.types.Object.sm64_obj_use_act6 = bpy.props.BoolProperty(name="Act 6", default=True)
 
+    # write to level script out
+    bpy.types.Object.sm64_segment_loads = bpy.props.PointerProperty(type=SM64_SegmentProperties)
```

Comment:
please add this propgroup to `SM64_ObjectProperties`

then access like `obj.fast64.sm64.segment_loads` instead of `obj.sm64_segment_loads`